Source/WebCore/platform/graphics/GraphicsLayer.h

```diff
@@ -545,7 +545,7 @@ class GraphicsLayer : public RefCountedAndCanMakeWeakPtr<GraphicsLayer> {
     WEBCORE_EXPORT virtual void suspendAnimations(MonotonicTime);
     WEBCORE_EXPORT virtual void resumeAnimations();
 
-    virtual Vector<std::pair<String, double>> acceleratedAnimationsForTesting(const Settings&) const { return { }; }
+    virtual Vector<std::tuple<String, double, bool>> acceleratedAnimationsForTesting() const { return { }; }
```

Comment:
Maybe it's time to make a struct for this return value.

Reply:
This is being done in https://github.com/WebKit/WebKit/pull/54238.